README.md

```diff
@@ -1770,6 +1770,8 @@ When constructing a Context, you receive two values - the context itself, along
 The returned cancel function can also be invoked in order to cancel an in-flight request.
 Note also that by using the `context.Context` instance with the `<operation-name>WithContext` method,
 the timeout applies only to a single operation invocation.
+
+By default the client has no request timeout, but a 30s connect timeout is set by default.
```

Comment:
I think this working might be a little confusing. Are you trying to document that the underlying client package we use doesn't have a default but that the SDK core is setting a default? Is it important to clarify what the underlying client package's default behavior is? Perhaps I'm missing (or forgetting) some context as to the value of thatclarification